src/styles/main.css

```diff
@@ -527,3 +527,7 @@ textarea.input {
 
 /* https://tailwindcss.com/docs/extracting-components#extracting-css-components-with-apply */
 @tailwind utilities;
+
+.elm-select-menu .elm-select-item:last-of-type .items-center > .border-dotted{
+  border-style: unset
+}
```

Comment:
please add an empty line at the end of the file

Reply:
sure! updated